precompile/crazy_with_power.go

```diff
@@ -0,0 +1,208 @@
+// This file is a generated precompile contract with stubbed abstract functions.
+// The file is generated by a template. Please inspect every code and comment in this file before use.
+
+// There are some must-be-done changes waiting in the file. Each area requiring you to add your code is marked with CUSTOM CODE to make them easy to find and modify.
+// Additionally there are other files you need to edit to activate your precompile.
+// These areas are highlighted with comments "ADD YOUR PRECOMPILE HERE".
+// For testing take a look at other precompile tests in core/stateful_precompile_test.go
+
+/* General guidelines for precompile development:
+1- Read the comment and set a suitable contract address in precompile/params.go. E.g:
+	CrazyWithPowerAddress = common.HexToAddress("ASUITABLEHEXADDRESS")
+2- Set gas costs here
+3- It is recommended to only modify code in the highlighted areas marked with "CUSTOM CODE STARTS HERE". Modifying code outside of these areas should be done with caution and with a deep understanding of how these changes may impact the EVM.
+Typically, custom codes are required in only those areas.
+4- Add your upgradable config in params/precompile_config.go
+5- Add your precompile upgrade in params/config.go
+6- Add your solidity interface and test contract to contract-examples/contracts
+7- Write solidity tests for your precompile in contract-examples/test
+8- Create your genesis with your precompile enabled in tests/e2e/genesis/
+9- Create e2e test for your solidity test in tests/e2e/solidity/suites.go
+10- Run your e2e precompile Solidity tests with 'E2E=true ./scripts/run.sh'
+
+*/
+
+package precompile
+
+import (
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ava-labs/subnet-evm/accounts/abi"
+	"github.com/ava-labs/subnet-evm/vmerrs"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
+	BenevelontGodGasCost uint64 = 20_000 // SET A GAS COST HERE
+	WreakHavocGasCost    uint64 = 20_000 // SET A GAS COST HERE
+
+	// CrazyWithPowerRawABI contains the raw ABI of CrazyWithPower contract.
+	CrazyWithPowerRawABI = "[{\"inputs\":[],\"name\":\"benevelontGod\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"},{\"inputs\":[],\"name\":\"wreakHavoc\",\"outputs\":[],\"stateMutability\":\"nonpayable\",\"type\":\"function\"}]"
+)
+
+// CUSTOM CODE STARTS HERE
+// Reference imports to suppress errors from unused imports. This code and any unnecessary imports can be removed.
+var (
+	_ = errors.New
+	_ = big.NewInt
+	_ = strings.NewReader
+	_ = fmt.Printf
+)
+
+// Singleton StatefulPrecompiledContract and signatures.
+var (
+	_ StatefulPrecompileConfig = &CrazyWithPowerConfig{}
+
+	CrazyWithPowerABI abi.ABI // will be initialized by init function
+
+	CrazyWithPowerPrecompile StatefulPrecompiledContract // will be initialized by init function
+)
+
+// CrazyWithPowerConfig implements the StatefulPrecompileConfig
+// interface while adding in the CrazyWithPower specific precompile address.
```

Comment:
not sure this comment makes sense? (adding in the CrazyWithPower specific precompile config? functionality?). maybe this should be updated in the generator?